_config.yml

```diff
@@ -7,7 +7,6 @@ twitter_username:
 github_username: 
 headings:
   news: 'News'
-  source: 'Source Code'
```

Comment:
why tho?

Reply:
Initially, I needed a less complicated way to map titles for pages that didn't have one without having to look at `headings` and try to map it based on the file name, or part of url. I noticed that pages like `/authors` worked fine because they had `primary_title` in their front matter; I added `primary_title` to the `source.markdown` file. However, since that was only place that `headings.source` was used, there was no longer a need to keep it.

Reply:
OK. I think the original idea was to have only one place to have to internationalize, but if that's not how we ended up involving in practice, I guess I can roll with it.